src/test/resources/application.conf

```diff
@@ -1,6 +1,8 @@
 auth-test-vars {
-    auth-url: "https://auth.sphere.io/oauth/token"
+    auth-url: "https://auth.sphere.io"
     auth-url: ${?AUTH_URL}
+    api-url: "https://api.sphere.io"
+    api-url: ${?API_URL}
```

Comment:
You should mention this new variable in https://github.com/commercetools/ctp-smoke-tests/blob/master/README.md

Reply:
done